data/categories.txt

```diff
@@ -2814,7 +2815,7 @@ highway-primary|highway-primary_link|highway-residential|highway-secondary|highw
 en:street|st|str|ave|road|rd
 ru:улица|переулок|проезд|тракт|проспект|ул
 uk:вулиця|провулок|проїзд|проспект|бульвар|вул
-de:Strasse|Weg|Avenue|Gasse|str
+de:Strasse|Weg|Avenue|Gasse|Str
```

Comment:
There is some kind of duplication, you're right. Seems that we should parse somehow this file to create a list of synonyms and use it during search.